tests/Invoke-DbaDbMirroring.Tests.ps1

```diff
@@ -15,23 +15,17 @@ Describe "$commandname Unit Tests" -Tag "UnitTests" {
 
 Describe "$commandname Integration Tests" -Tag "IntegrationTests" {
     BeforeAll {
-        $null = Get-DbaProcess -SqlInstance $TestConfig.instance2 | Where-Object Program -Match dbatools | Stop-DbaProcess -Confirm:$false -WarningAction SilentlyContinue
```

Comment:
these have been important previously due to resource constraints and killing processes was our only way to assure results. we should keep an eye out for future issues.

Reply:
In case we need this in a test - its the fault of some other test. The goal should be to identify the other test. I already identified some cases and in an "interactive" test environment like my Azure lab it is possible to find them.